src/index.js

```diff
@@ -53,9 +53,9 @@ const ForgotPassword = React.lazy(() => import('Components/Accounts/ForgotPasswo
 const Order = React.lazy(() => import('Components/Order/Order'))
 const WhiteLabelSEO = React.lazy(() => import('Pages/WhiteLabelSEO'))
 const Profile = React.lazy(() => import('Pages/Profile'))
-const PriceComparison = React.lazy(() => import('Components/Home/PriceComparison/PriceComparison'))
-const Team = React.lazy(() => import('Components/Home/Team/Team'))
-const About = React.lazy(() => import('Components/Home/About/About'))
+// const PriceComparison = React.lazy(() => import('Components/Home/PriceComparison/PriceComparison'))
```

Comment:
@IntegralTeam why is it commented out?

Reply:
Because we don't need to lazyload it. Simply, we should load all Homepage components at startup, because they're neccesary.